test/e2e/editors/textEditor.spec.js

```diff
@@ -224,7 +224,7 @@ describe('TextEditor', () => {
     }, 200);
   });
 
-  it('should hide whole editor when it is higher then header', (done) => {
+  it('should hide whole editor when it is higher then header and TD is not rendered anymore', (done) => {
```

Comment:
Please use
```
() => async() {
   ...
   await sleep(...);
   ...
}
```
instead of
```
() =>(done) {
   ...
   setTimeout(() => {
      done();
   }, ...);
   ...
}
```